ansible/roles/ml-analytics-adhoc-jobs/defaults/main.yaml

```diff
@@ -0,0 +1,2 @@
+BASEPATH: "/opt/managelearnjobs"
+USER: "data-pipeline"
```

Comment:
@raghupathiguduri is data-pipeline user created as part of provision/server bootstrapping scripts?